src/test/java/org/vandeseer/easytable/util/PdfUtilTest.java

```diff
@@ -11,13 +11,27 @@
 
 public class PdfUtilTest {
 
+
+    /**
+     * The delta that is still acceptable in float comparisons.
+     */
+    private static final double EPSILON = 0.0001;
+
+
     @Test
     public void getStringWidth_noNewLines() {
         final String text = "this is a small text";
 
-        assertThat(PdfUtil.getStringWidth(text, PDType1Font.HELVETICA, 12), equalTo(94.692F));
+        assertThat(isEqualInEpsilon(94.692F, PdfUtil.getStringWidth(text, PDType1Font.HELVETICA, 12)), is(true));
+    }
+
+    private static boolean isEqualInEpsilon(float a, float b) {
```

Comment:
We should make this package visible in `PdfUtil` instead of duplicating.

Reply:
But it's a bad practise to use methods of the tested class to check if the output correct.  

It's like: "Hey, I will test you and you say me if you're passed."

Reply:
But I found a better solution. :sunglasses: